modules/primer-utilities/docs/borders.md

```diff
@@ -109,7 +109,7 @@ Use `border-dashed` to give an element a dashed border.
 
 ## Rounded corners
 
-Add or remove rounded corners: `rounded-0` removes rounded corners, `rounded-1` applies a border radius of 3px, `rounded-2` applies a border radius of 6px.
+Add or remove rounded corners: `rounded-0` removes rounded corners, `rounded-1` applies a border radius of 3px, `rounded-2` applies a border radius of 6px. `.circle` applies a border radius of 50%, which turns square elements into perfect circles.
```

Comment:
I realize this wasn't your edit, but could we start with a slightly nicer intro, such as:

> Use the following utilities to add or remove rounded corners: